docs/developer/release-cycle.md

```diff
@@ -108,6 +108,6 @@ style: [Kubernetes 1.26 Release Team](https://github.com/kubernetes/sig-release/
 
 | **Role**                                  | **Lead Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
 |-------------------------------------------|---------------------------------------|----------------------------------------|
-| Release Lead                              |                                       |                                        |
-| Communications/Docs/Release Notes Manager |                                       |                                        |
-| CI Signal/Bug Triage/Automation Manager   |                                       |                                        |
+| Release Lead                              |  Yuvraj Kakaraparthi([@ykakarap](https://github.com/ykakarap)/Slack:`@Yuvaraj Kakaraparthi`)  | Jim Ntosas([@dntosas](https://github.com/dntosas)/ Slack:`@dntosas`), Carlos Panato([cpanato](https://github.com/cpanato)/ Slack:`@cpanato`), Amit Kumar([@hackeramitkumar](https://github.com/hackeramitkumar)/ Slack:`@Amit Kumar`) |
```

Comment:
By comparing the Slack ID, seems the Release lead name is incorrect? @ykakarap might confirm

Reply:
Great catch! :bow:

Reply:
Thanks for catching this!